apps/docs/src/common/pages/styles/typography.tsx

```diff
@@ -31,8 +31,8 @@ const Page: FC<PageProps> = ({ location }) => (
 
       <span className="govuk-caption-xl">Styles</span>
       <h1 className="heading-section">Typography</h1>
-      <p className="lede">GDS Transport, the typeface used on GOV.UK, is restricted by license for use on gov.uk domains. Internal services can't use GDS Transport and should instead use the open source <A href="https://rsms.me/inter/">Inter UI</A> typeface.</p>
-      <p className="lede">Inter UI is closely related to Helvetica and Arial but is easier to read online, particularly at smaller type sizes.</p>
+      <p className="lede">GDS Transport, the typeface used on GOV.UK, is restricted by license for use on gov.uk domains. Internal services can't use GDS Transport and should instead use the open source <A href="https://fonts.google.com/specimen/Roboto#standard-styles">Roboto</A> typeface.</p>
+      <p className="lede">Roboto is closely related to Helvetica and Arial but is optimised for screens, making it easier to read online.</p>
```

Comment:
I wonder if it would be more accurate to say 'similar to'.

@karypun: Any thoughts?